packages/salesforcedx-vscode-core/test/jest/telemetry/index.test.ts

```diff
@@ -5,8 +5,9 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import { TELEMETRY_GLOBAL_USER_ID } from '@salesforce/salesforcedx-utils-vscode/src/constants';
```

Comment:
is the import here intentional? 
@salesforce/salesforcedx-utils-vscode/src/constants -> @salesforce/salesforcedx-utils-vscode

Reply:
Yes

Reply:
is that value not exported from the top of the package? 

I realize this is a test, but this a pattern that should be avoided as it causes bundling issue when outside of tests.